data/weapons/weapon_data.csv

```diff
@@ -275,7 +275,7 @@ Micro Glaux,diableavionics_zephyr_rightBeam,2,0.5,3000,650,300,,,,75,8,3,1,,ENER
 ,diableavionics_zephyr_rightRod,,,,1,1,,,,,,,,,,,,,10,,,,,,,,,,,,,,SYSTEM,,,,,,,,,,,,,,,
 ,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 #HOAR,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-Burchel (long delay),diableavionics_hoar_head,,,1500,400,200,,,,36,9,6,0.75,2,ENERGY,,,0,0,0.48,0.02,,,,,1000000,,,,,,,"PD,PD_ONLY,SYSTEM",,,,,,,,,,,,,,,
+Burchel (long delay),diableavionics_hoar_head,,,1500,600,,150,,,36,9,9,0.75,2,ENERGY,,,0,0,0.48,0.02,,,,,1000000,,,,,,,"PD,PD_ONLY,SYSTEM",,,,,,,,,,,,,,,
```

Comment:
Hoar Burchel (long delay) range set to 600 (from 400), damage set to 150 (from 200), ammo max set to 9 (from 6)